modules/compliance-operator-cli-installation.adoc

```diff
@@ -22,8 +22,11 @@ kind: Namespace
 metadata:
   labels:
     openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/enforce: privileged <1>
```

Comment:
Is this only rendered in the 4.12 documentation? I don't think older versions of OCP require this.

Reply:
So, I have this against our main branch in case we need it in future releases too - but it will be cherrypicked to 4.12 only.